Vance Leon

vanceleon opened the pull request.

vanceleon committed:
first commit

vanceleon committed:
Answers to questions

vanceleon committed:
additions to html doc

vanceleon committed:
Desktop view

vanceleon committed:
changes for tablet view

vanceleon committed:
minor changes to desktop,
addition to have mobile view

vanceleon committed:
15 minute left commit

vanceleon committed:
committing

vanceleon committed:
commit

mikemarshal commented:
1st and 2nd media queries looking nice Vance, and I love that you are making so many commits!

Just need to fix the last media query objective and some of the styling

vanceleon committed:
adding untracked commit

vanceleon committed:
fixed the tablet view
added the solution css file to the css folder
renamed the design files to open in order

vanceleon committed:
Fixed major issues with viewport
updated desktop
updated tablet
updated mobile view

vanceleon committed:
desktop and tablet complete
working on mobile view

vanceleon committed:
latest css

vanceleon committed:
the latest version with nav and text breakpoint errors

vanceleon committed:
MVP for desktop tablet and mobile breakpoints

vanceleon committed:
MVP
